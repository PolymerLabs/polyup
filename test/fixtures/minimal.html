<link rel='import' href='../polymer/polymer.html'>

<polymer-element name='minimal' noscript></polymer-element>
