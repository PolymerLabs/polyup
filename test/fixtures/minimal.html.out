<link rel="import" href="../polymer/polymer.html">

<dom-module id="minimal">
</dom-module>
<script>
  Polymer({ is: 'minimal' });
</script>
